index.html

```diff
@@ -0,0 +1,119 @@
+    <style>
+      body,
+      p {
+        font-family: "Roboto", sans-serif;
+      }
+    </style>
+    <link href="http://fonts.cdnfonts.com/css/crete-round" rel="stylesheet" />
+
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <section class="header">
+      <div>
+        <p class="logo">John Doe</p>
+      </div>
+      <div class="menu_btn">
+        <div class="menu"></div>
+        <div class="menu"></div>
+        <div class="menu"></div>
+      </div>
+    </section>
+    <section class="headline">
+      <p class="text_primary">Hey There. I’m Alex</p>
+      <p class="text_supporting">I’am a Software Developer</p>
+      <p class="text_description">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <div class="icons">
+        <svg
+          width="18"
+          height="20"
+          viewBox="0 0 18 20"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M17.9989 7.46114C17.9989 6.32124 17.6329 5.25043 16.9342 4.31779C17.2004 3.41969 17.2669 2.27979 17.2004 0.967185C17.1671 0.414508 16.7345 0 16.2022 0C15.9028 0 13.3741 0.0345423 11.9101 1.38169C10.6458 1.1399 9.31495 1.1399 8.01736 1.38169C6.58667 0.0345423 4.05802 0 3.7253 0C3.19296 0 2.76042 0.414508 2.72715 0.967185C2.62734 2.27979 2.72715 3.41969 2.99333 4.31779C2.29462 5.28497 1.92863 6.35579 1.92863 7.46114C1.92863 9.8791 3.7253 12.0553 6.45358 13.0915C6.35377 13.2988 6.28723 13.5406 6.22068 13.7824C3.32604 13.4715 1.9619 10.7427 1.89536 10.639C1.66246 10.1209 1.06357 9.91364 0.56449 10.19C0.0654144 10.4318 -0.134216 11.0535 0.131958 11.5717C0.198501 11.7444 2.02845 15.4404 6.05432 15.8549V18.9637C6.05432 19.5509 6.48686 20 7.05248 20H12.875C13.4406 20 13.8732 19.5509 13.8732 18.9637V14.8532C13.8732 14.2314 13.7401 13.6442 13.5072 13.1261C16.2022 12.0553 17.9989 9.91364 17.9989 7.46114Z"
+            fill="#DBD8D7"
+          />
+        </svg>
+        <svg
+          width="20"
+          height="20"
+          viewBox="0 0 20 20"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M13.6439 3.03442C10.1551 3.03442 7.28809 5.86201 7.28809 9.37925V19.3103C7.28809 19.6896 7.59897 19.9999 7.97893 19.9999H11.5713C11.9513 19.9999 12.2622 19.6896 12.2622 19.3103V9.65511C12.2622 8.89649 12.8839 8.2758 13.6439 8.2758C14.4038 8.2758 15.0256 8.89649 15.0256 9.65511V19.3103C15.0256 19.6896 15.3364 19.9999 15.7164 19.9999H19.3088C19.6888 19.9999 19.9997 19.6896 19.9997 19.3103V9.37925C19.9997 5.89649 17.1672 3.03442 13.6439 3.03442Z"
+            fill="#DBD8D7"
```

Comment:
- [ ]  To make your code more professional, I think you should implement the list of items by using the `ul` tag to the group and each element inside a `li` . Add also a working link to each icons with `a` tag for each element.